blog/2024-03-22-renku-2-0/index.mdx

```diff
@@ -0,0 +1,110 @@
+it.
+
+We are calling this new iteration “Renku 2.0” even though Renku version 1.0 does
+not currently exist! We want to imprint the idea that this is a new concept, not
+just an evolution of the current one.
+
+## Mix and match: code, data and compute
+
+To enable flexible use cases, Renku 2.0 is based on modular project
+configurations. Users can bootstrap their projects with a mix of code
+repositories, data sources, and compute environments and evolve
+them as the project matures. For example, in an early stage of a project, a data
+scientist may simply require a quick way to connect a cloud compute environment
+to a data source for some basic data exploration; as the analysis of the data
+deepens, they may need to add a code repository to start versioning the code and
+collaborate with others in a more structured way. Later on, someone else on the
+project might build a containerized app or dashboard to demonstrate the data
+properties and model results more easily with non-experts.
+
+Renku 2.0 will allow you to select the components that you need for your project
+to work. Beyond the current capabilities that Renku already offers, you can
+levereage the flexibility of Renku 2.0 in multiple situations, for example:
+
+- If you are testing out RenkuLab... and you want to bring your **existing GitHub repo** into a project and start a compute session from **the docker image you have already defined**.
+- If you are starting a new project... and you want to **initialize a git repository** and **start out with a basic compute environment**. Later, you will decide what data and storage you need.
+- If you find an interesting dataset... and you want to **test it out in a live compute environment** so you can decide quickly if you want to move forward with it.
+- If you are working on a project that uses **multiple repositories** from multiple collaborators... and you want to use all of them in 1 Renku project together.
+- If you need to showcase my project... and you want to create a **dashboard that uses a different environment configuration than the development environment**.
+
+A “Renku project” is where data science work gets done. For this, a few ingredients are required, nammely:
+
+- Data
+- Code
+- Compute Environment
+
+Renku’s role is about making it easy for researchers to **mix and match** the elements they need to get their work done. In addition, Renku makes these elements **searchable**, encourages their **reuse** between projects, and **illuminates connections** between them (even across Projects) wherever needed. Renku accomplishes this by extracting and indexing useful metadata about each of those elements.
+
+### Data
+
+Data science projects need data! Users can connect different types of cloud data
+sources to their projects, like S3 buckets. Depending on the nature of the
+storage, they may be used in read-only or in read-write mode. However, projects
```

Comment:
I think the read/write functionality is an unnecessary level of detail for this post. 
```suggestion
sources to their projects, such as S3 buckets and WebDAV resources. Projects
```